packages/server/src/entity/NotifModel.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  BaseEntity,
+  ManyToOne,
+  JoinColumn,
+} from "typeorm";
+import { UserModel } from "./UserModel";
+
+@Entity("notif_model")
+export class NotifModel extends BaseEntity {
```

Comment:
This naming makes it confusing whether this is the model for a single notification, or a representation of a device to which you can notify a given user. Maybe call it "NotifEndpoint" or "NotifDevice" or just "Device"?

Reply:
i agree. in #34 we changed it to DesktopNotifModel or something